src/main/java/org/tikv/raw/SmartRawKVClient.java

```diff
@@ -30,6 +30,7 @@
 import org.tikv.service.failsafe.CircuitBreaker;
 
 public class SmartRawKVClient implements RawKVClientBase {
+
```

Comment:
Why add this empty line? Is it caused by the formater?

Reply:
Yes